pkg/reconciler/clusterimagepolicy/clusterimagepolicy.go

```diff
@@ -260,6 +269,32 @@ func (r *Reconciler) inlinePolicies(ctx context.Context, cip *v1alpha1.ClusterIm
 			return err
 		}
 	}
+	if cip.Spec.Policy != nil && cip.Spec.Policy.URL != nil {
+		err := r.inlinePolicyURL(ctx, cip.Spec.Policy)
+		if err != nil {
+			logging.FromContext(ctx).Errorf("Failed to read policy url %s: %v", cip.Spec.Policy.URL.String(), err)
+			return err
+		}
+	}
+	return nil
+}
+
+func (r *Reconciler) inlinePolicyURL(ctx context.Context, policyRef *v1alpha1.Policy) error {
+	logging.FromContext(ctx).Infof("inlining policy url %q", policyRef.URL.String())
+	resp, err := http.Get(policyRef.URL.String())
```

Comment:
Perhaps we can set a pre-defined timeout to avoid waiting long time for a response.

```
var client = &http.Client{
  Timeout: time.Second * 10,
}
response, _ := client.Get(policyRef.URL.String())
```